CLDConfig/Tracking/ConformalTracking.py

```diff
@@ -0,0 +1,103 @@
+#
+# Copyright (c) 2014-2024 Key4hep-Project.
+#
+# This file is part of Key4hep.
+# See https://key4hep.github.io/key4hep-doc/ for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from Gaudi.Configuration import WARNING
+from Configurables import MarlinProcessorWrapper
+
+
+if any(small_vtx in geoservice.detectors[0] for small_vtx in ["_o2_", "_o3_", "_o4_"]):
```

Comment:
`geoservice` is just happily known because whatever magic SequenceLoader does?

Reply:
Yes, it comes from here:
https://github.com/key4hep/CLDConfig/blob/11bf30217e95fc1d6d0a583f4836cee5321f1c2f/CLDConfig/CLDReconstruction.py#L68-L71

I agree that it is a bit ugly that this is not obvious from the loaded files